docs/fidesops/mkdocs.yml

```diff
@@ -18,10 +18,18 @@ nav:
           - Managing Datastores: ui/datastores.md
           - User Management: ui/user_management.md
   - How-To Guides:
+      - Application Configuration Reference: guides/configuration_reference.md
+      - View Available Connection Types: guides/connection_types.md
+      - Connect to SQL and NoSQL Databases: guides/database_connectors.md
+      - Configure Datasets: guides/datasets.md
+      - Configure Email Communications: guides/email_communications.md
+      - Authenticate with Oauth: guides/oauth.md
```

Comment:
A section of these docs were reordered to lines 7-12 and don't need to be listed twice, they just need to be rehomed in the project directory. 

It looks like the Connection Type guide is new? 🤔

Reply:
ohhh I see, yup, looks like just the connection type guide wasn't listed in the nav before. It's relatively new, added July 6th 2022. I'll go ahead and remove the others from this `guides` section.